src/com/palmergames/bukkit/towny/Towny.java

```diff
@@ -851,4 +852,16 @@ else if (Bukkit.getServer().getName().equalsIgnoreCase("craftbukkit")) {
 		
 		metrics.addCustomChart(new Metrics.SimplePie("resident_uuids_stored", () -> TownySettings.getUUIDPercent()));
 	}
+	
+	public static boolean is116Plus() {
+		String verString = Bukkit.getBukkitVersion();
+		verString = verString.replace("-R0.1-SNAPSHOT", "");
+		
+		Version ver = new Version(verString);
+		Version required = new Version("1.16.1");
+		
+		TownyMessaging.sendErrorMsg(ver.compareTo(required) + "");
```

Comment:
Just caught this one. This will broadcast an error message any time its run on non-1.16+, right? If so, it will have to be removed as well.

Reply:
Yeah, it was left there on accident.